sufia-models/app/actors/sufia/generic_file/actor.rb

```diff
@@ -102,11 +107,22 @@ def save_and_record_committer
         return false unless generic_file.save
       rescue RSolr::Error::Http => error
         ActiveFedora::Base.logger.warn "Sufia::GenericFile::Actor::save_and_record_committer Caught RSOLR error #{error.inspect}"
+        Sufia.config.upload_logger.warn "Sufia::GenericFile::Actor::save_and_record_committer Error saving #{generic_file.title}. Caught RSOLR error #{error.inspect}"
+        save_tries += 1
+        # fail for good if the tries is greater than 3
+        raise error if save_tries >= 3
+        sleep 0.01
+        retry
+      rescue Ldp::Gone => error
+        Sufia.config.upload_logger.warn "Sufia::GenericFile::Actor::save_and_record_committer Error saving #{generic_file.title}. Caught LDP error #{error.inspect}"
         save_tries += 1
         # fail for good if the tries is greater than 3
         raise error if save_tries >= 3
         sleep 0.01
         retry
+      rescue StandardError => error
+        Sufia.config.upload_logger.warn "Sufia::GenericFile::Actor::save_and_record_committer Error saving #{generic_file.title}. Caught Exception error #{error.inspect}"
```

Comment:
I have no problem consolidating.  I'm not sure why Solr error was singled out for retry.  Now that I understand the meaning of the Ldp::Gone error, it doesn't make sense to do a retry for that.  So the other option is to retry for Solr errors and simply log for all others.